src/components/experience/ExperienceItem.tsx

```
@@ -0,0 +1,39 @@
+				<p className="text-neutral-400 light:text-neutral-700">
+					at <span className="font-semibold">{company.title}</span>,{" "}
+					{company.description}
+				</p>
+			</div>
+
+			<hr className="border-neutral-800 lg:hidden" />
+
+			<div className="flex max-w-prose flex-col gap-8 lg:col-span-2">
+				<ul className="flex flex-col gap-4 text-neutral-400 light:text-neutral-700">
+					{tasks.map((task) => (
+						<li>{task}</li>
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)